src/Misc/MiddleWare.cpp

```diff
@@ -2067,6 +2080,14 @@ void MiddleWareImpl::handleMsg(const char *msg)
     } else {
         //printf("Message Handled<%s:%s>...\n", msg, rtosc_argument_string(msg));
     }
+
+    // now handle all chained messages
+    while(!msgsToHandle.empty())
+    {
+        std::vector<char> front = msgsToHandle.front();
```

Comment:
looks right, though I always have to check the docs since C++ is the only lang I've used where queue.pop() doesn't return the value that was popped.

Reply:
Yeah, I checked the `std::queue` docs carefully because I wasn't sure, too. But this one is right.